app.js

```diff
@@ -12,14 +12,45 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
+hbs.registerPartials(path.join(__dirname, 'views/partials/'));
+
 // Register the location for handlebars partials here:
 
 // ...
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render(__dirname + '/views/index');
```

Comment:
since there is a code line in L11 to set the views in the __dirname + / views you don't need to redeclare the path here, just by putting index or index.hbs it should locate and render the correct hbs.